### PR TITLE
[jsk_perception] Add depth image filter

### PR DESCRIPTION
**Pull request by iory**
# What is this?

Create a mask image from the distance of a depth image.

image

### CODE REVIEW

File: doc/jsk_perception/nodes/depth_image_filter.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+  Maks image.
+
+
+Parameters
+----------
+
+* ``~threshold`` (Float, Default: ``0.0``)
+
+  Mask threshold. Normally, ixels with a depth value farther than this value will be masked.
```
Comment (708yamaguchi) [inline doc/jsk_perception/nodes/depth_image_filter.rst:34]: ixels -> pixels?

File: doc/jsk_perception/nodes/depth_image_filter.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  Input depth image.
+
+
+Publishing Topic
+----------------
+
+* ``~output/mask`` (``sensor_msgs/Image``)
+
+  Maks image.
```
Comment (708yamaguchi) [inline doc/jsk_perception/nodes/depth_image_filter.rst:26]: Maks -> Mask

**Review by 708yamaguchi — APPROVED** [review]
Thank you.
Very interesting feature.

I can easily use this PR with Fetch camera.
threshold:=1.0
Screenshot 2021-12-21 18:14:34

**Comment by 708yamaguchi** [comment]
Sorry, I mistakenly pressed the approve button yesterday, but please fix typos.

**Comment by iory** [comment]
Thanks! Fixed it.

**Review by 708yamaguchi — APPROVED** [review]

**Comment by 708yamaguchi** [comment]
Thank you for your quick response!

**Pull request merged**